### ISSUE TITLE
skipExtract

### ISSUE THREAD

**Issue by betterapp** [BlackbitDigitalCommerce/pimcore-data-director#94]
As I see there is extra param skipExtract but I can not find anything about it in DD documentation.

![image](https://github.com/BlackbitDigitalCommerce/pimcore-data-director/assets/7449124/7388ce62-ead3-409c-b3f5-7314d9620c33)

**Comment by betterapp** [comment]
When I use skipExtract=1 it looks like it take all rows data
and it took 3x more time then the same query without this param

for what skipExtract param is used ? Why we have difference in data count ?

![image](https://github.com/BlackbitDigitalCommerce/pimcore-data-director/assets/7449124/67f1b6f7-44be-45b3-9e39-6feaa3c9b2b7)

**Comment by betterapp** [comment]
As I checked skipExtract=1 param causes to read data from tables:

![image](https://github.com/BlackbitDigitalCommerce/pimcore-data-director/assets/7449124/476a91f1-e958-46ca-a20c-d5973fcafbf3)

And looks like it read ALL rows from this tables for specific dataport.

What I see is that when we Delete row data and try to run API call which should export data we got:

No dataport resource IDs given

in the response and DataDirector create empty process
![image](https://github.com/BlackbitDigitalCommerce/pimcore-data-director/assets/7449124/4e330bf1-2c51-42b2-9e00-3877ca1ed7bf)

that will run forever. Need to be canceled manually.

I checked it on 3.2.35 DD.
